python/python/tests/tests_server_modules/test_json.py

```diff
@@ -624,3 +624,224 @@ async def test_json_nummultby(self, glide_client: TGlideClient):
         # Check for Overflow in legacy
         with pytest.raises(RequestError):
             await json.nummultby(glide_client, key, ".key9", 1.7976931348623157e308)
+
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_json_debug(self, glide_client: TGlideClient):
```

Comment:
nice testing, but keep in mind, we're testing the client, not the module.

Reply:
Thanks.
do you have any suggestions? or do you think that the tests were too comprehensive?

Reply:
> too comprehensive

It isn't bad, but you can save your time, effort and nerves in future